relayer/src/lib.rs

```diff
@@ -8,7 +8,7 @@
     unused_qualifications,
     rust_2018_idioms
 )]
-// #![allow(dead_code, unreachable_code, unused_imports, unused_variables)]
+#![allow(clippy::large_enum_variant)]
```

Comment:
I wonder if we shouldn't move this annotation to the site where it's necessary, namely here:

https://github.com/informalsystems/ibc-rs/blob/e3ccb5055306f15016403e5bd08a8e91393680b2/relayer/src/chain/handle.rs#L53